src/Conf.ml

```diff
@@ -741,6 +742,39 @@ let config =
     Arg.(
       value & opt list_assoc default & info ["c"; "config"] ~doc ~docs ~env)
 
+let apply_janestreet_profile c =
+  { c with
+    break_cases= `Fit
+  ; break_infix= `Fit_or_vertical
+  ; break_string_literals= `Wrap
+  ; doc_comments= `Before
+  ; extension_sugar= `Preserve
+  ; field_space= `Loose
+  ; if_then_else= `Keyword_first
+  ; indicate_nested_or_patterns= false
+  ; infix_precedence= `Parens
+  ; leading_nested_match_parens= true
+  ; let_and= `Sparse
+  ; let_open= `Preserve
+  ; margin= 90
+  ; ocp_indent_compat= true
+  ; parens_tuple= `Multi_line_only
+  ; sequence_style= `Terminator
+  ; type_decl= `Sparse
+  ; wrap_fun_args= false
+  ; module_item_spacing= `Compact }
+
+let profile =
+  let doc = "Preset profiles." in
+  let names = ["p"; "profile"] in
+  let all =
+    [ ("default", Fn.id, "$(b,default) uses the default values of options.")
```

Comment:
The default profile should set all value to the default values, rather than be a no-op.

Reply:
Similar for the janestreet profile.